### ISSUE TITLE
Yarn installations

### ISSUE THREAD

**Issue by zanettin** [axemclion/browser-perf#86]
Hi

When we install `browser-perf` via Yarn, the "cleanup modules" tasks cleans up `docs` folder which is required that the code works as expected. Maybe you can rename the folder to ensure correct behaviour also for Yarn users?

Thanks a lot for this great tool!
Cheers

**Comment by axemclion** [comment]
@zanettin Thanks for reporting the issue. 

Are you using `yarn` clean and running into the issue ? Could you please help with repro steps ?

**Comment by zanettin** [comment]
Hi @axemclion 

Yes exactly. `yarn clean` does that.
Love this task and works quite great except for `browser-perf` on our react app 😄 
What do you think?

Seams like this directories are removed:
```
docs
doc
website
images
assets
```

**Comment by axemclion** [comment]
@zanettin  Do you suggest I rename docs to something else ? I could look to see if there is a way to not delete those folders.

**Comment by zanettin** [comment]
yes,  if this is an option for you. listed folders from above are deleted by yarn clean. if you find another workaround/option it would also be great :-) thx
